supertokens_python/recipe/thirdpartypasswordless/utils.py

```diff
@@ -193,12 +208,41 @@ def get_email_delivery_config(
             ev_feature = email_verification_feature
             email_service = BackwardCompatibilityService(recipe.app_info, tppless_recipe, create_and_send_custom_email, ev_feature)
 
-        if email_delivery_config is not None and email_delivery_config.override is not None:
-            override = email_delivery_config.override
+        if email_delivery is not None and email_delivery.override is not None:
+            override = email_delivery.override
         else:
             override = None
 
         return EmailDeliveryConfigWithService(email_service, override=override)
 
-    return ThirdPartyPasswordlessConfig(override=OverrideConfig(functions=override.functions, apis=override.apis), providers=providers, contact_config=contact_config, flow_type=flow_type, get_link_domain_and_path=get_link_domain_and_path, get_custom_user_input_code=get_custom_user_input_code, email_verification_feature=validate_and_normalise_email_verification_config(recipe, email_verification_feature, override),
-                                        get_email_delivery_config=get_email_delivery_config)
+    def get_sms_delivery_config() -> SMSDeliveryConfigWithService[TypeThirdPartyPasswordlessSmsDeliveryInput]:
+        if sms_delivery and sms_delivery.service:
+            return SMSDeliveryConfigWithService(
+                service=sms_delivery.service,
+                override=sms_delivery.override
+            )
+
+        if isinstance(contact_config, (ContactPhoneOnlyConfig, ContactEmailOrPhoneConfig)):
+            pless_create_and_send_custom_text_message = contact_config.create_and_send_custom_text_message
+        else:
+            pless_create_and_send_custom_text_message = None
```

Comment:
this check defers so much from how it's done in passwordless' utils file - this is better than doing that assert thing and checking based on equality with "EMAIL". Please change the passwordless utils thing to do it this way as well.

Reply:
Yeah. Done.